02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,66 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Ben's Site</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+  <div>
+<img src="https://avatars2.githubusercontent.com/u/6634962?v=3&s=460"/>
+  </div>
+
+  <div>
+<h1 style="font-family: cursive">Benjamin Blowe</h1>
+  </div>
+<p>Born here in San Antonio, spent my childhood living in Florida, and have been back in SATX for the last decade. I'm an anthropology student at UTSA, I currently work in customer service, and I'm here at ACA to make a career in web development.</p>
+
+  <div>
+<h4>Contact:</h4>
+<h4>bdbblowe@gmail.com | (210) 487-8118</h4>
+</div>
+
+  <div>
+<img src="http://docs.whitesourcesoftware.com/download/attachments/17989744/github_logo.png?version=1&modificationDate=1463491374000" alt="Github">
+</div>
+
+  <div>
+    <h2 style="font-family: cursive">Hobbies</h2>
+  </div>
+
+  <div>
+    <ul>
+      <li>Coding</li>
+      <br></br>
+      <li>Reading</li>
+      <br></br>
+      <li>Going to concerts</li>
+      <br></br>
+      <li>Walking my dogs</li>
+    </ul>
+  </div>
+
+  <div style="float: left; margin-left:150px;">
+       <h3 style="font-family: cursive;  text-align: left">Favorite News Sites</h3>
+
+  <table style="float: left; text-align: left">
+  <tr>
+    <td><a href="https://news.vice.com/"</href>VICE NEWS</a></td>
+    <td><a href="https://www.youtube.com/user/journeymanpictures"</href>Journeyman Pictures</a></td>
+  </tr>
+  <tr>
+    <td><a href="https://roarmag.org/"</href>ROAR Magazine</a></td>
+    <td><a href="https://www.opendemocracy.net/"</href>openDemocracy</a></td>
+  </tr>
+</table>
+</div>
+<div style="float: right; margin-right:150px;">
+    <h3 style="font-family: cursive">Favorite TV Shows</h3>
+<ol type="I">
+  <li><a href="https://www.youtube.com/watch?v=oQ46aGDO3HY"</href>Mr. Robot</a></li>
```

Comment:
Use the `target="_blank"` attribute for your <a> tags to open those website in a separate tab or window, this way the visitor of your website will still remain on your website. This is what you want, right?